components/automate-ui/src/app/services/http/http-client-auth.interceptor.ts

```diff
@@ -29,20 +34,19 @@ export class HttpClientAuthInterceptor implements HttpInterceptor {
     let headers = request.headers.set('Authorization', `Bearer ${this.chefSession.id_token}`);
     const filtered = request.params.get('unfiltered') !== 'true';
     // Uncomment here and after to clone() arg list
-    // after https://github.com/angular/angular/issues/18812 is fixed.
+    // after we've upgraded to angular 7.2+ (for this issue:
+    // https://github.com/angular/angular/issues/18812).
     // const params = request.params.delete('unfiltered');
     if (this.projects && filtered) {
       headers = headers.set('projects', this.projects);
     }
     return next
-      .handle(request.clone({
-        headers
-      })).pipe(
-        catchError((response: HttpEvent<any>) => {
-          if (get('status', response) === 401) {
```

Comment:
This `get('status', response)` was a wonky way to side-step type checks, I believe. `HttpEvent<any>` doesn't have a status propery, but it's a type union that contains `HttpErrorResponse`. By using the more specific type (the latter) here, we can call `response.status` below.